src/Data/Vec/Properties.agda

```diff
@@ -155,6 +156,41 @@ take-drop-id (suc m) (x ∷ xs) = begin
     x ∷ xs
   ∎
 
+--------------------------------------------------------------------------------
+-- truncate
+
+truncate-refl : ∀ {n} → (xs : Vec A n) → truncate ≤-refl xs ≡ xs
+truncate-refl []       = refl
+truncate-refl (x ∷ xs) = cong (x ∷_) (truncate-refl xs)
+
+truncate-trans : ∀ {m n p} → (le₁ : m ≤ n) → (le₂ : n ≤ p) → (xs : Vec A p) →
```

Comment:
Any chance we could give the inequalities in these proofs more meaningful names, i.e. `m≤n` rather than `le₁`? Don't forget to update the CHANGELOG as well!